irods/test/helpers.py

```diff
@@ -212,14 +245,15 @@ def make_flat_test_dir(dir_path, file_count=10, file_size=1024):
             f.write(os.urandom(file_size))
 
 @contextlib.contextmanager
-def create_simple_resc (self, rescName = None, vault_path = ''):
+def create_simple_resc (self, rescName = None, vault_path = '', hostname = ''):
     if not rescName: 
         rescName =  'simple_resc_' + unique_name (my_function_name() + '_simple_resc', datetime.datetime.now())
     created = False
+    host_name = self.sess.host if not hostname else hostname
     try:
         self.sess.resources.create(rescName,
                                    'unixfilesystem',
-                                   host = self.sess.host,
+                                   host = host_name,
```

Comment:
Unless GitHub is playing tricks on me (as it has in the past) it seems that the one-liner suggested here (https://github.com/irods/python-irodsclient/pull/458/files/#r1256208203) has been reverted somehow...

Reply:
Hmm, don't know how it happened, but yep.  Correcting now.